.github/workflows/render-all.yml

```diff
@@ -184,15 +205,17 @@ jobs:
 
       # Run Coursera version
       - name: Convert Leanpub quizzes to Coursera
-        if: needs.yaml-check.outputs.toggle_leanpub == 'yes'
+        if: needs.yaml-check.outputs.toggle_leanpub == 'yes' && needs.yaml-check.outputs.toggle_quiz_check == 'yes'
```

Comment:
This step for coursera is really only needed if there's quizzes AND if Leanpub is being published.